packages/terra-menu/tests/wdio/menu-spec.js

```diff
@@ -6,21 +6,21 @@ Terra.describeViewports('Menu', ['medium'], () => {
   it('displays a default Menu', () => {
     browser.url('/raw/tests/cerner-terra-framework-docs/menu/menu/default-menu');
     $('#default-button').click();
-    $('[class*="content"][aria-modal="true"][role="dialog"]').isFocused();
+    $('li:first-child[role="menuitem"]').isFocused();
```

Comment:
It seems like this is just returning a boolean without any operation (see [the doc](https://webdriver.io/docs/api/element/isFocused/) for isFocused); can we update this to use an `expect` similar to [this](https://github.com/cerner/terra-core/blob/35cbecb6941c298631f9eabd0dd3fe1795622a55/packages/terra-show-hide/tests/wdio/show-hide-spec.js#L25)?

Reply:
Updated in https://github.com/cerner/terra-framework/pull/1626/commits/0cef7c5c71a4099facd6dc55678d76c6cfc43424